lib/timecop/time_extensions.rb

```diff
@@ -45,36 +45,31 @@ def today_with_mock_date
     alias_method :strptime_without_mock_date, :strptime
 
     def strptime_with_mock_date(str = '-4712-01-01', fmt = '%F', start = Date::ITALY)
-      unless start == Date::ITALY
-        raise ArgumentError, "Timecop's #{self}::#{__method__} only " +
-          "supports Date::ITALY for the start argument."
-      end
-
       #If date is not valid the following line raises
-      Date.strptime_without_mock_date(str, fmt)
+      Date.strptime_without_mock_date(str, fmt, start)
 
       d = Date._strptime(str, fmt)
       now = Time.now.to_date
       year = d[:year] || now.year
       mon = d[:mon] || now.mon
       if d.keys == [:year]
-        Date.new(year)
+        Date.new(year, 1, 1, start)
```

Comment:
I don't love hardcoding the 1s here or repeating all the occurrences of `start` but this seemed safer than refactoring the method.